### PR TITLE
[ST-823] Remove existing snippets

### PR DESCRIPTION
**Pull request by Soichiro725**
### Purpose/goal of PR (Issue #)
https://wovnio.atlassian.net/browse/ST-823

Remove existing snippets.

### Comments
Any delimiter can be used in PHP.
By using `@`, no need to escape `/`.

### CODE REVIEW

File: src/wovnio/html/HtmlConverter.php [hunk 1]
```diff
@@ -127,15 +128,27 @@ private function replaceDom($dom, &$marker)
...
-        $html = $this->removeTagFromHtmlByRegex($html, $snippet_regex);
-
+        $html = $this->remoceSnippet($html);
         $snippet_code = $this->buildSnippetCode($add_fallback_mark);
         $parent_tags = array("(<head\s?.*?>)", "(<body\s?.*?>)", "(<html\s?.*?>)");
 
         return $this->insertAfterTag($parent_tags, $html, $snippet_code);
     }
 
+    private function remoceSnippet($html)
```
Comment (bstewart00) [inline src/wovnio/html/HtmlConverter.php:138]:
```suggestion
    private function removeSnippet($html)
```
small typo

Comment (Soichiro725) [reply src/wovnio/html/HtmlConverter.php:138]: Thanks!

**Review by zeyuwu — APPROVED** [review]
That's some impressive regex there.

**Review by bstewart00 — APPROVED** [review]
looks good

**Pull request merged**